From 033b7605bb64582b18a068e94e603ab772a9e85b Mon Sep 17 00:00:00 2001
From: Dan Kephart <dan.kephart@lairdtech.com>
Date: Thu, 26 Feb 2015 14:52:10 -0500
Subject: [PATCH] laird: ARM: wb45n: enable conntrack events

conntrack events allows for netfilter's connection tracker to send events
to userspace for debugging connection issues

Bug: 7068
---
 arch/arm/configs/wb45n_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/wb45n_defconfig b/arch/arm/configs/wb45n_defconfig
index d4e8359..380b947 100644
--- a/arch/arm/configs/wb45n_defconfig
+++ b/arch/arm/configs/wb45n_defconfig
@@ -67,6 +67,7 @@ CONFIG_NF_CONNTRACK=m
 CONFIG_NF_CONNTRACK_FTP=m
 CONFIG_NF_CONNTRACK_TFTP=m
 CONFIG_NF_CT_NETLINK=m
+CONFIG_NF_CONNTRACK_EVENTS=y
 CONFIG_NETFILTER_XTABLES=y
 CONFIG_NETFILTER_XT_MATCH_CONNTRACK=m
 CONFIG_NETFILTER_XT_MATCH_MAC=m
-- 
1.8.3.2

